ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -227,6 +228,9 @@ function pr-audit-build-android-run-queue(){
 
         # Remove build directory.
         rm -rf build_android_lib
+
```

Comment:
As we all know, this module is not common after this PR. It seems that the location of this module needs to be moved from plugins-**base**/pr-audit-build-android.sh to plugins-**staging**/pr-audit-build-android.sh

Reply:
How about calling the script `api/android/build-android-lib.sh` in `pr-audit-build-android.sh`
This would allow `pr-audit-build-android.sh` to be generic

Note that `api/android/build-android-lib.sh` will only do the API build. The NDK build will still be performed by `pr-audit-build-android.sh`.

Reply:
We may add patching mechanism in android build script of nnstreamer. (I think it's already in there: build-android-lib.sh)

Reply:
Yes, its already there
however, as per my understanding, that build script isn't currently used in the CI build
That script will have to updated with `pr-audit-build-android.sh`, and then use it in here

Reply:
@jaeyun-jung , PTAL, Coul you comment your opinion on this discussion?

Reply:
To run build-android-lib.sh, we have to define three variables
1. ANDROID_HOME: Android SDK
2. GSTREAMER_ROOT_ANDROID: GStreamer prebuilt libraries for Android
3. NNSTREAMER_ROOT: NNStreamer root directory

In AWS,
2. GSTREAMER_ROOT_ANDROID is set for ndk-build (shared object). We cannot use this because API library uses static prebuilt libraries.
3. NNSTREAMER_ROOT is needed to set per PR.

To avoid these issues. I made and uploaded a file ```gradle.properties`` which sets gstAndroidRoot and nnstreamerRoot.  (see line 212)

```
gstAndroidRoot=/var/www/html/android/gstreamer-1.0-android-universal-1.16.0
nnstreamerRoot=nnstreamer-path
```

Reply:
To use build-android-lib.sh in AWS, we have to update this script to parse the args for gstAndroidRoot and nnstreamerRoot.